File: data/conala-train_code-davinci-002_1666524963/Java/156.java
Map<String, Double> map = new HashMap<>();
map.put("a", 1.0);
map.put("b", 2.0);
map.put("c", 3.0);

double sum = 0.0;
for (Double value : map.values()) {
    sum += value;
}
double average = sum / map.size();

File: data/conala-train_code-davinci-002_1666524963/Python/156.py
# Create a dictionary
d = {'a': 1, 'b': 2, 'c': 3, 'd': 4, 'e': 5}

# Create a new dictionary with the same keys, but the values are the average of the values in the original dictionary
d_avg = {k: sum(v)/len(v) for k, v in d.items()}

# Print the new dictionary
print(d_avg)